### DISCUSSION TITLE
Long durations of hemodialysis sessions, ?should be classified as CVVH vs discarded

### DISCUSSION THREAD

**Discussion by cloverbunny** [MIT-LCP/mimic-code#1259] [MIMIC-IV]
Hi!
I was looking at some hemodialysis session duration information in MIMIC-IV, and noticed that some of the durations are extremely long. Usually, hemodialysis sessions are about four hours, so about 240 minutes, but some of these are thousands of minutes - example below. 
CVVH/CRRT is continuous dialysis and so would be more likely to fit with these really long sessions; is there any way to tell if these were incorrectly documented as HD when they should have been CRRT? Or would people just throw out these long values (except there are over 900 cases like this, would be a long of data lost)? It matters to me because I want to keep track of what days patients were on HD and what days they were on CRRT.
Thanks! 

Example long hemodialysis sessions (sorry, all the columns labels are shifted over by one)
 
  | subject_id | hadm_id | stay_id | starttime | endtime | itemid | value | valueuom | label | category | value_minutes
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
19004463 | 24357541 | 39101627 | 2177-01-30 10:03:00 | 2177-02-01 01:28:00 | 225441 | 2365 | min | Hemodialysis | 4-Procedures | 2365
15110026 | 28089871 | 38700117 | 2142-03-27 13:15:00 | 2142-03-28 16:59:00 | 225441 | 1664 | min | Hemodialysis | 4-Procedures | 1664
12199246 | 25482501 | 33794813 | 2179-07-28 11:07:00 | 2179-07-29 14:37:00 | 225441 | 1650 | min | Hemodialysis | 4-Procedures | 1650
17304212 | 29292229 | 38138468 | 2154-04-10 13:20:00 | 2154-04-14 10:09:00 | 225441 | 5569 | min | Hemodialysis | 4-Procedures | 5569
17810811 | 25877275 | 30529254 | 2138-10-12 21:30:00 | 2138-10-19 08:00:00 | 225441 | 9270 | min | Hemodialysis | 4-Procedures | 9270

Example CVVH sessions - there are some short ones, but that's reasonable clinically because sometimes the circuit clots and has to be taken down or sometimes the circuit has to be broken to travel etc  (sorry, all the columns labels are shifted over by one)

  | subject_id | hadm_id | stay_id | starttime | endtime | itemid | value | valueuom | label | category
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
11538389 | 22188993 | 37455091 | 2145-10-30 10:16:00 | 2145-11-04 06:59:00 | 225802 | 7003.0 | min | Dialysis - CRRT | Dialysis
11538389 | 22188993 | 37455091 | 2145-11-02 06:00:00 | 2145-11-02 14:00:00 | 225802 | 480.0 | min | Dialysis - CRRT | Dialysis
11538389 | 22188993 | 37455091 | 2145-11-02 15:10:00 | 2145-11-02 19:50:00 | 225802 | 280.0 | min | Dialysis - CRRT | Dialysis
11538389 | 22188993 | 37455091 | 2145-11-02 21:00:00 | 2145-11-03 00:30:00 | 225802 | 210.0 | min | Dialysis - CRRT | Dialysis
11538389 | 22188993 | 37455091 | 2145-11-03 02:30:00 | 2145-11-03 10:30:00 | 225802 | 480.0 | min | Dialysis - CRRT | Dialysis
11538389 | 22188993 | 37455091 | 2145-11-03 11:10:00 | 2145-11-04 07:02:00 | 225802 | 1192.0 | min | Dialysis - CRRT | Dialysis
11538389 | 22188993 | 37455091 | 2145-11-05 14:57:00 | 2145-11-06 03:15:00 | 225802 | 738.0 | min | Dialysis - CRRT | Dialysis
11538389 | 22188993 | 37455091 | 2145-11-06 12:45:00 | 2145-11-07 01:03:00 | 225802 | 738.0 | min | Dialysis - CRRT | Dialysis
11538389 | 22188993 | 37455091 | 2145-11-07 03:00:00 | 2145-11-07 13:00:00 | 225802 | 600.0 | min | Dialysis - CRRT | Dialysis

**Comment by cloverbunny** [comment]
@alistairewj - sorry to tag you directly, wondering if you have any thoughts? Thanks so much!

**Comment by alistairewj** [comment]
Well it's hard to say as there's no fixed rules on documentation of procedures in MetaVision. When nursing staff document a procedure like this, it results in a duration bar in a view in the front end (basically a Gantt chart for ongoing procedures). It's not a consistently used aspect of the interface, so this might be the result of nursing staff documenting data for different purposes - I don't know the specifics but you could imagine using this for "this is a dialysis patient" rather than "we are actively performing dialysis". I created a few derived views for hemodialysis and I ignore the *procedureevents* data, instead using the documentation of hemodialysis output in *outputevents* to capture IHD. For CRRT, I consistently found using the charted data better resolved the actual active dialysis (e.g. I could account for pauses due to line clots, etc). I did less comprehensive assessment of intermittent dialysis, mainly because there's only one real piece of information usable for it: the hemodialysis output. So for IHD, I lose all duration information potentially documented in *procedureevents*. In contrast, not all dialysis sessions are documented in *procedureevents*, but pretty much every one is documented in *outputevents* as they want an accurate fluid balance calculation.

So no fixed answer here, but food for thought.

**Reply by fysqy** [reply]
Does outputevents include the patient's dialysis records in non-ICU ward?
